andela/AndelaEatsUI
# #163876985 create dashboard mini reports for the admin

#### What does this PR do?
- Adds a dashboard and a mini report to the dashboard for vendor performance

#### Description of Task to be completed?

- create a component for the vendor performance report
- create action and reducer to get data from rest endpoint
- write unit tests for the above components

#### How should this be manually tested?

- check out to this branch 
- run `npm install`
- the left menu item should have the dashboard active by default 
- vendor performance graph should be visible on the dashboard 

#### What are the relevant pivotal tracker stories?
- [#163876985](https://www.pivotaltracker.com/story/show/163876985)

#### Gif 
![eatsdashboard](https://user-images.githubusercontent.com/30045448/55380503-11abc280-5529-11e9-9098-d4a879ff6137.gif)
